spec/models/plant_accession_spec.rb

```diff
@@ -47,6 +47,25 @@
     end
   end
 
+  context 'unique attributes' do
+    it 'disallows duplicate plant_accession and originating_organisation' do
```

Comment:
@kammerer The **scoped** uniqueness test fails because it attempts to insert a null reference record into the DB (to facilitate the test), with only the relevant field (`plant_accession`) filled in. Such a record is not, however, valid because of other constraints defined for the `plant_accession` table. More specifically, when the spec is changed to:
```
    it { should validate_uniqueness_of(:plant_accession).scoped_to(:originating_organisation) }
```
...the following happens:
```
  1) PlantAccession unique attributes should require case sensitive unique value for plant_accession scoped to originating_organisation
     Failure/Error: it { should validate_uniqueness_of(:plant_accession).scoped_to(:originating_organisation) }

     ActiveRecord::StatementInvalid:
       PG::CheckViolation: ERROR:  new row for relation "plant_accessions" violates check constraint "plant_accessions_pub_chk"
       DETAIL:  Failing row contains (a, null, null, null, null, null, null, null, null, null, null, null, null, null, null, 1, 0, 2016-08-16 08:26:44.372272, 2016-08-16 08:26:44.372272, null, t, null, null).
       : INSERT INTO "plant_accessions" ("plant_accession", "created_at", "updated_at") VALUES ($1, $2, $3) RETURNING "id"
```

I googled around and I haven't really been able to figure out how to make `validate_uniqueness_of` work with a FactoryGirl-generated reference object rather than a blank record. My guess is, `shoulda_matchers` is just not clever enough. ;) Perhaps you can suggest a way to proceed - otherwise I'll follow @Nuanda 's suggestion and just use `update_column`.

On the other hand, the **unscoped** uniqueness tests work correctly.